app/logic/emailHandler.py

```diff
@@ -0,0 +1,89 @@
+from flask import Flask, request, render_template
+import yaml, os
+from flask_mail import Mail, Message
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+from app.models.emailTemplate import EmailTemplate
+from app import app
+import sys
+from pathlib import Path
+
+def load_config(file):
+    """ Loads the 'default' file in the config directory. """
+    with open(file, 'r') as ymlfile:
+        cfg = yaml.load(ymlfile, Loader=yaml.FullLoader)
+    return cfg

```

Comment:
The configuration is already loaded in celts. We don't want to copy in LSF's worse configuration handling

Reply:
removed it